### PR TITLE
fix: mobile sign out

### LINKED ISSUE #4582: [Mobile] The option to Sign Out and the account email info are no longer available on homepage
**[Affected Versions]:**
- Safari 14.6
- Chrome 92

**[Affected Platforms]:**
- Google Pixel
- Iphone SE 2

**[Prerequisites]:**
- You are logged into "https://hubs.mozilla.com/".

**[Steps to reproduce]:**
1. Navigate to https://hubs.mozilla.com/.
2. Observe if the option to Sign Out and if the logged in account email info are displayed.

**[Expected results]:**
- The option and the email account are displayed.

**[Actual results]:**
- Only the option to Sign In/Sign Up is displayed.

**[Notes]:**
- The issue is not reproducible within the More menu inside a Hubs room.
- This issue can confuse already logged in users and also new users by giving the impression that the login was not successful.
- Attached is a screen recording of the issue: [link](https://drive.google.com/file/d/14V7cv3YFbEdffPwvE_SFIBiftPFyMV2V/view?usp=sharing).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-769)

### PR DESCRIPTION
**Pull request by matthewbcool**
adds mobile sign-out to resolve #4582

### CODE REVIEW

File: src/react-components/home/HomePage.js [hunk 1]
```diff
@@ -50,12 +51,27 @@ export function HomePage() {
...
 
   const canCreateRooms = !configs.feature("disable_room_creation") || auth.isAdmin;
-
+  const email = auth.email;
   return (
     <PageContainer className={styles.homePage}>
       <Container>
         <div className={styles.hero}>
-          <SignInButton mobile />
+          {auth.isSignedIn ? (
```
Comment (netpro2k) [inline src/react-components/home/HomePage.js:59]: If we find ourselves copying this code some more we might want to turn it into a component that handles both logged in and out states.

File: src/react-components/home/HomePage.scss [hunk 2]
```diff
@@ -35,8 +35,16 @@
 
 :local(.sign-in-container) {
```
Comment (netpro2k) [inline src/react-components/home/HomePage.scss:36]: I was confused about re-using this classname, but I don't actually see where it was being used before...

**Review by netpro2k — COMMENTED** [review]
I would double check that `sign-in-container` isn't being used anywhere else, but otherwise LGTM

File: src/react-components/home/HomePage.scss (hunk 2 above)
Comment (matthewbcool) [reply src/react-components/home/HomePage.scss:36]: I think it was some unused styling before.

**Pull request merged**